### ISSUE TITLE
Pikmin 4?

### ISSUE THREAD

**Issue by Enterprise12nx01** [StevensND/switch-port-mods#2]
Would it be possible to make a pikmin 4 60 fps patch? seems to be hard as nobody has done it but figured i would try asking! Ty!

**Comment by StevensND** [comment]
**The mod exists** and **[you can download it here](https://drive.google.com/drive/folders/1oP6NYTxJggJhxxGrtTVIKQbE5CpbHMEc)** but it has issues (I don't know all of them as I'm not a Pikmin fan and the game has never caught my attention). Nor have I found a list of all the issues detected.

Some of the issues that I know are:

> Ingame cutscenes are sped up. And there are some other bugs such as pikmin not being spawn and such. These issues aren't fixed yet with the current mod

[Info taken from this video
](https://youtu.be/0IfFrgtQfk0?si=EbkwtZyRMpYugw5c)

**Fixing speedups is not an easy task.** You have to test and see if the fixes you make don't break other things. I also know that some things are hardcoded and makes it more harder.

For this reason, a properly working mod has not yet been made for this game. Besides, I don't see anyone who makes mods interested in it either. Maybe one day when all the problems are known in detail someone will take an interest.

**Comment by StevensND** [comment]
[60FPS v.1.0.2 Cheat WIP.zip](https://github.com/user-attachments/files/16040965/60FPS.v.1.0.2.Cheat.WIP.zip)
[60FPS v.1.0.2 WIP.zip](https://github.com/user-attachments/files/16040966/60FPS.v.1.0.2.WIP.zip)

You can also try this mod and cheat and see if it has the same issues or not.

**Comment by Enterprise12nx01** [comment]
> [60FPS v.1.0.2 Cheat WIP.zip](https://github.com/user-attachments/files/16040965/60FPS.v.1.0.2.Cheat.WIP.zip) [60FPS v.1.0.2 WIP.zip](https://github.com/user-attachments/files/16040966/60FPS.v.1.0.2.WIP.zip)
> 
> You can also try this mod and cheat and see if it has the same issues or not.

Yeah I've tried the original mod but it's pretty broken. I will try these, are they different in some way?
The original mod the most notable issues were Pikmin would constantly get lost following you/disband and cutscenes were sped up. I played with it for awhile and those were the only issues I really noticed.  I'm sure there's a few other issues but I never ran into any

**Comment by StevensND** [comment]
> Yeah I've tried the original mod but it's pretty broken. I will try these, are they different in some way? The original mod the most notable issues were Pikmin would constantly get lost following you/disband and cutscenes were sped up. I played with it for awhile and those were the only issues I really noticed. I'm sure there's a few other issues but I never ran into any

I don't know the method they used for the first mod. This one I made was improvised using some of my knowledge so I don't know if there is a difference or not.

Also, I haven't played enough to know or identify all the possible issues.

**Comment by StevensND** [comment]
Any update on this?

**Comment by Enterprise12nx01** [comment]
> Any update on this?

so far in the quick test, cutscenes are still broken but pikmin dont seem to get lost while following you anymore!
i wasnt usually able to get 60 fps, about 45 even with my 5800x3d and 7900xtx though. i had both the cheat and mod active

**Comment by StevensND** [comment]
> > Any update on this?
> 
> so far in the quick test, cutscenes are still broken but pikmin dont seem to get lost while following you anymore! i wasnt usually able to get 60 fps, about 45 even with my 5800x3d and 7900xtx though. i had both the cheat and mod active

You shouldn't have both things enable at the same time. Just enable 1: The mod or the cheat but not both.

Regarding the FPS: That's weird cause I have a GTX 1080, Ryzen 5600 and 16GB RAM and I have 60FPS stable.

In which resolution are you playing?. Also I'd suggest you to clean your shaders.

Regarding the cutscenes: maybe I can fix it using something from the DBZ Kakarot patch.

Keep testing it anyway. Thanks.

**Comment by Enterprise12nx01** [comment]
Ok I'll try the mod only and report back. I'm using Ryujinx, 1440p, I'll try messing around with settings

**Comment by StevensND** [comment]
> Ok I'll try the mod only and report back. I'm using Ryujinx, 1440p, I'll try messing around with settings

Any update?

**Comment by Enterprise12nx01** [comment]
> > Ok I'll try the mod only and report back. I'm using Ryujinx, 1440p, I'll try messing around with settings
> 
> Any update?

sorry havent tested, was going to wait to see if you find any cutscene fix

**Comment by Enterprise12nx01** [comment]
Any update on a fix for cutscenes?

**Comment by StevensND** [comment]
> Any update on a fix for cutscenes?

I haven't continued testing. When I have time I will continue.

**Comment by Enterprise12nx01** [comment]
> > Any update on a fix for cutscenes?
> 
> I haven't continued testing. When I have time I will continue.

ok ty!

**Comment by lxf5466** [comment]
Do you have a 60-frame patch for the atmospheric system? If so, can you send it? I would be very grateful.

**Comment by StevensND** [comment]
> Do you have a 60-frame patch for the atmospheric system? If so, can you send it? I would be very grateful.

I don't know what you're talking about @lxf5466 lol
